### PR TITLE
tests/main/services-install-hook-can-run-svcs: add variant w/o --enable

### PR DESCRIPTION
**Pull request by anonymouse64**
This test currently fails on master.

### CODE REVIEW

**Review by pedronis — APPROVED** [review]
thanks

**Review by stolowski — APPROVED** [review]
Thank you!

File: tests/main/services-install-hook-can-run-svcs/task.yaml [hunk 1]
```diff
@@ -1,6 +1,15 @@
 summary: Check that install hooks in snaps can start services
 
+environment:
+  FLAGS/enable: --enable
+  FLAGS/none: ""
```
Comment (pedronis) [inline tests/main/services-install-hook-can-run-svcs/task.yaml:5]: now that I looked at test logs  none is not very self-explanatory here,  s/none/one-off/ ? or something, anyway I'm going to land this but maybe on master we should follow-up with a better naming

Comment (anonymouse64) [reply tests/main/services-install-hook-can-run-svcs/task.yaml:5]: What about "no-flag" ?

**Comment by pedronis** [comment]
That's fine as well. My main opposition to "none" in this context is that
it's very opaque.

**Pull request merged**